data_models/asana_data_model.py

```diff
@@ -0,0 +1,15 @@
+import panther_event_type_helpers as event_type
+
+audit_log_type_map = {
+    "user_login_succeeded": event_type.SUCCESSFUL_LOGIN,
+    "user_login_failed": event_type.FAILED_LOGIN,
+    "user_invited": event_type.USER_ACCOUNT_CREATED,
+    "user_reprovisioned": event_type.USER_ACCOUNT_CREATED,
+    "user_deprovisioned": event_type.USER_ACCOUNT_DELETED,
+    "user_workspace_admin_role_changed": event_type.ADMIN_ROLE_ASSIGNED,
+}
+
+
+def get_event_type(event):
+    match = audit_log_type_map.get(event.get("event_type")) #None if not matched
```

Comment:
Not sure I understand the comment.  Is there ever a chance the `event_type` can be none? If so, this will error, right?

Reply:
This was a poorly worded comment. I was indicating that this was a safe dict get so the function would return None implicitly instead of a key error. I changed the wording